lib/google_crawler/search/keyword_query.ex

```diff
@@ -0,0 +1,44 @@
+defmodule GoogleCrawler.Search.KeywordQuery do
+  import Ecto.Query
+  alias GoogleCrawler.Repo
+  alias GoogleCrawler.Search.{Keyword, KeywordSearchForm}
+
+  def all(user, params) do
+    Keyword
+    |> where([k], k.user_id == ^user.id)
+    |> filter_by_term(params)
+    |> filter_by_links_count(params)
+    |> order_by(desc: :inserted_at)
+    |> Repo.all()
```

Comment:
Should we move Repo command out and let the query only have the queries? We can do the Repo.all in context instead 🤔

Reply:
Fixed